service/kvmconfig/framework.go

```diff
@@ -130,10 +130,7 @@ func NewFramework(config FrameworkConfig) (*framework.Framework, error) {
 			RandomkeysSearcher: randomkeysSearcher,
 
 			GuestUpdateEnabled: config.GuestUpdateEnabled,
-			HandledVersionBundles: []string{
-				"1.1.0",
-			},
-			Name: config.Name,
+			Name:               config.Name,
```

Comment:
The relationship between the resource version and the version bundle version is now more straight. It should be harder to configure a wrong version bundle version for a resource version with this approach since everything is kept in one place.